examples/basics/visuals/grid_mesh.py

```diff
@@ -0,0 +1,53 @@
+"""
+Demonstration of a mesh constructed in a grid about x,y,z coordinates.
+"""
```

Comment:
I think in our other examples that require `scipy` we make a note of it at the top, can you `git grep scipy` to find them and do whatever we did in those here?